openedx/core/djangoapps/user_api/api/profile.py

```diff
@@ -91,36 +87,6 @@ def profile_info(username):
 
 
 @intercept_errors(ProfileInternalError, ignore_errors=[ProfileRequestError])
-def update_profile(username, full_name=None):
```

Comment:
@wedaly FYI that this PR also includes some deletion of your version of profile updates.